advisories/unreviewed/2022/05/GHSA-x534-j49x-mqvj/GHSA-x534-j49x-mqvj.json

```diff
@@ -1,19 +1,43 @@
 {
   "schema_version": "1.4.0",
   "id": "GHSA-x534-j49x-mqvj",
-  "modified": "2023-03-01T18:30:58Z",
+  "modified": "2023-03-09T05:05:46Z",
   "published": "2022-05-24T16:57:50Z",
   "aliases": [
     "CVE-2019-11932"
   ],
-  "details": "A double free vulnerability in the DDGifSlurp function in decoding.c in libpl_droidsonroids_gif before 1.2.15, as used in WhatsApp for Android before 2.19.244, allows remote attackers to execute arbitrary code or cause a denial of service.",
+  "summary": "Double free in android-gif-drawable",
+  "details": "A double free vulnerability in the DDGifSlurp function in decoding.c in the android-gif-drawable library before version 1.2.18, as used in WhatsApp for Android before version 2.19.244 and many other Android applications, allows remote attackers to execute arbitrary code or cause a denial of service when the library is used to parse a specially crafted GIF image.",
```

Comment:
General question: Are changes from NVD not synced back here? According to https://nvd.nist.gov/vuln/detail/cve-2019-11932#VulnChangeHistorySection this change already happened in 2019.

(Though the NVD entry has some issues, it contains multiple duplicate reference links.)